src/SFA.DAS.AANHub.Application.UnitTests/CalendarEvents/Queries/GetCalendarEventsQueryHandlerTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AANHub.Application.CalendarEvents.Queries;
+using SFA.DAS.AANHub.Domain.Entities;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+using SFA.DAS.AANHub.Domain.Models;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.AANHub.Application.UnitTests.CalendarEvents.Queries;
+public class GetCalendarEventsQueryHandlerTests
+{
+    [Test]
+    [RecursiveMoqAutoData]
+    public async Task Handle_CalendarEventsNotFound_ReturnsEmptyList(
+        [Frozen] Mock<ICalendarEventsReadRepository> calendarEventsReadRepositoryMock,
+        GetCalendarEventsQueryHandler sut,
+        Member member,
+        CancellationToken cancellationToken
+        )
+    {
+        calendarEventsReadRepositoryMock.Setup(c => c.GetCalendarEvents(member.Id, cancellationToken))
+            .ReturnsAsync(() => new List<CalendarEventSummary>());
+
+        var result = await sut.Handle(new GetCalendarEventsQuery(member.Id, 1), new CancellationToken());
+
+        result.Result.CalendarEvents.Count.Should().Be(0);
+        result.Result.TotalCount.Should().Be(0);
+        result.Result.PageSize.Should().Be(Domain.Common.Constants.CalendarEvents.PageSize);
```

Comment:
Why are we asserting on pagesize here?

Reply:
Just to check the model is being set based on the constant.

Reply:
removed